interface/modules/custom_modules/oe-module-faxsms/src/Controller/AppDispatch.php

```diff
@@ -34,7 +34,7 @@ abstract class AppDispatch
     protected $_currentAction;
     protected $credentials;
     private $_request, $_response, $_query, $_post, $_server, $_cookies, $_session;
-    private $authUser;
+    protected $authUser;
```

Comment:
This propery is a work in progress to remove as I was cleaning up when adding VoiceService and deciding if I wanted to let service have separate credentials and in the end decide to keep with standard credentials.
After I reviewed you can remove as I think I've got it covered. If not leave protected and I'll remove next time in that module. thanks